core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/JsonCodec.scala

```diff
@@ -111,6 +112,14 @@ object JsonCodec {
     "errorCode",
     "timestamp"
   )(x => RuntimeError.unapply(x).get)
+  implicit val diskStatusEncoder: Encoder[DiskStatus] = Encoder.encodeString.contramap(_.toString)
+  implicit val diskAuditInfoEncoder: Encoder[DiskAuditInfo] = Encoder.forProduct4(
```

Comment:
this looks about the same as existing AuditInfo, can we reuse the same case class?

Reply:
That would be nice… I noticed that the existing `AuditInfo` had 1 field that's not relevant to disks. Will see if I can refactor so we can reuse it.

Reply:
I'm running into the same thing with kubernetes right now. I think creator, createdDate, destroyedDate are the core fields between all 3 use cases